templates/index.html

```diff
@@ -66,8 +66,24 @@ <h2>{{recipe['name']}}</h2>
         <!-- <h2>Sushi</h2>
```

Comment:
Remove temporary/sample text once integration of search functionality results into index.html is complete

Reply:
will implement these specific changes in next push!